code/renderers/react/src/docs/config.ts

```diff
@@ -8,6 +8,9 @@ export const parameters = {
     story: { inline: true },
     extractArgTypes,
     extractComponentDescription,
+    source: {
+      excludeDecorators: true,
+    },
```

Comment:
@tmeasday Should I set it to true for the other renderers as well? Because in the code we assume that it is set per default to true. E.g. here: https://github.com/storybookjs/storybook/blob/valentin/fix-source-code-view-for-nextjs/code/renderers/react/src/docs/jsxDecorator.tsx#L210

Reply:
I think this is a question that @shilman might need to answer.